uml-editor/src/models/jointjs/uml-classifier/Note-Element.ts

```diff
@@ -0,0 +1,119 @@
+import { dia, shapes, util } from '@joint/core'
+import { CustomJointJSElementAttributes } from '../custom-jointjs-element.model'
+export const NoteElementView = dia.ElementView.extend({
+  events: {
+    'input input': 'onInput',
+  },
+  onInput: function (event: dia.Event) {
+    this.model.attr('text/props/value', event.target.value)
+  },
+})
+
+const initialWidth = 150
+const initialHeight = 80
+export class NoteElement extends shapes.standard.Rectangle {
+  private backgroundColor = '#f5e487'
+  override readonly markup = [
```

Comment:
do a const with the markup and import it then in the Class like in the cleanup branch